### DISCUSSION TITLE
Cloud ENV.Dart

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5767] [New Feature/Ideas Requests] [answered]
It would be nice and functional to have a system where we can remotely intervene in the env dart file as in the design file.

**Comment by chungxon** [comment, marked as answer]
Hi all,

You can use Firebase Remote config feature to update env.dart file remotely.
Ref: https://docs.fluxbuilder.com/firebase-remote-config

Regards,
